cfgov/unprocessed/css/pages/bureau-structure.less

```diff
@@ -12,88 +12,27 @@
 @node-border-width: 3px;
 @node-border-color: @gray-20;
 
+
 .org-chart {
     margin-top: @margin;
     background: @gray-10;
     border-top:1px solid @node-border-color;
 
+    .u-hide-unless-mobile {
```

Comment:
`.u-show-on-mobile` already [exists in cf-core](https://github.com/cfpb/capital-framework/blob/canary/src/cf-core/src/cf-utilities.less#L262). Use that instead.